raiden/network/transport/matrix.py

```diff
@@ -202,9 +201,9 @@ def start(
 
         # TODO: Add greenlet that regularly refreshes our presence state
         self._client.set_presence_state(UserPresence.ONLINE.value)
+        self._send_queued_messages()  # uses property instead of initial_queues
 
-        gevent.spawn_later(1, self._send_queued_messages, queueids_to_queues)
-        self.log.info('TRANSPORT STARTED')
+        self.log.info('TRANSPORT STARTED', config=self._config)
```

Comment:
Is it possible to have private information in the config dictionary? If it is then it must not be logged.

Reply:
No, no private information is contained on it, only things like timeouts and current server (which is already logged all over the place)